### DISCUSSION TITLE
[Enhancement] [select-v2] SleectV2 item高度 设置

### DISCUSSION THREAD

**Discussion by Zbuter** [element-plus/element-plus#9701] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select-v2`

## Description

在el-select 中可以通过 修改 el-option 样式对select中的item高度进行定义， 但在el-select-v2中 并没有找到控制item的高度的方法。

### Additional comments

_(empty)_

**Comment by YujiaCheng1996** [comment]
tree-v2也需要！
